ci-operator/step-registry/cucushift/installer/check/ibmcloud/byok/cucushift-installer-check-ibmcloud-byok-commands.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# IBM Cloud CLI login
+function ibmcloud_login {
+    export IBMCLOUD_CLI=ibmcloud
+    export IBMCLOUD_HOME=/output
+    region="${LEASED_RESOURCE}"
+    rg=$1
+    export region
+    echo "Try to login to ${rg}..."
+    "${IBMCLOUD_CLI}" login -r ${region} -g ${rg} --apikey @"${CLUSTER_PROFILE_DIR}/ibmcloud-api-key"
+}
+
+RESOURCE_GROUP=$(jq -r .ibmcloud.resourceGroupName ${SHARED_DIR}/metadata.json)
+
+ibmcloud_login ${RESOURCE_GROUP}
+
+critical_check_result=0
+
+kp_file=${SHARED_DIR}/ibm_kp.json
+
+if [ -f ${kp_file} ]; then
+    id=$(jq -r .id ${kp_file})
+else
+    echo "[ERROR] fail to get the key protect instance id !!"
+    exit 1
+fi
+
+#check the kpKey whether used in the volumes of the nodes(master & worker)
+mapfile -t vols < <(ibmcloud kp registrations -i ${id} -o JSON  | jq -r .[].resourceCrn)
+echo "INFO: key registrations list is: ${#vols[@]}" "${vols[@]}"
+
+#check that node os disk is encrypted
+machines=$(oc get machines -A --no-headers | awk '{print $2}')
+
+for machine in ${machines}; do
+    echo "--- check machine ${machine} ---"
+    volCrn=$(ibmcloud is instance ${machine} --output JSON | jq -r .boot_volume_attachment.volume.crn)
```

Comment:
Do you have the real value of the `volCrn`?

Reply:
Your log only printed out key registrations list, I am asking `volCrn` here.

Reply:
https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/46712/rehearse-46712-periodic-ci-openshift-verification-tests-master-installer-rehearse-4.15-installer-rehearse-ibmcloud-byok/1735664575740645376